tools/telemetry/www/semantic.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
It's the semantic.ui configuration file, used when building the semantic/dist/ files from source. I'm going to add this in the file.